source/opt/CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ add_library(SPIRV-Tools-opt
   types.h
   type_manager.h
   unify_const_pass.h
+  value_number_table.h
+        local_redundancy_elimination.h
```

Comment:
odd formatting

Reply:
Fixed, plus I moved it into the correct place.  That is what happens you trust the tools too much.